src/css/components/_text-with-graphic.scss

```diff
@@ -0,0 +1,34 @@
+.text-with-graphic-template {
+    width: 100%;
+    height: calc(#{$master-height} - 75px);
+    padding: 25px;
+    display: flex;
+}
+
+.text-with-graphic-container {
+    height: 100%;
+    width: 50%;
+    display: flex;
+    align-items: center;
+}
+
+.text-with-graphic-template .large-graphic {
+    padding-right: 25px;
+    padding-left: 10px;
+    height:75%;
+    object-fit: contain;
+
```

Comment:
Don't know why it removed my comment, but please remove extra newline (same with _graphic-with-text.scss)